pyproject.toml

```
@@ -1,10 +1,101 @@
 [build-system]
-requires = ["pip==21.3.1", "setuptools==59.6.0", "setuptools-scm[toml]>=6.2"]
+requires = ["pip==22.3.1", "setuptools==65.6.3", "setuptools-scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pymanopt"
```

This seems necessary to get `setuptools-scm` to work.
```suggestion
name = "pymanopt"
dynamic = ["version"]
```